app/modules/listings/components/listing_editor.svelte

```diff
@@ -99,10 +107,25 @@
     use:autosize
   />
 </label>
-<VisibilitySelector
-  bind:visibility
-  showTip={true}
-/>
+<div class="visibility-selector">
+  <VisibilitySelector
+    bind:visibility
+    showTip={true}
+  />
+</div>
+<label>
```

Comment:
I think the listing type selector should be placed before the name input field, as that's really determining the nature of the list and should thus be considered first by users creating a list